dapps/marketplace/src/components/Balances.js

```diff
@@ -37,6 +37,7 @@ const Balances = ({ currencies, title, className, onClose }) => {
                   className="more-ogn"
                   onClose={onClose}
                   onNavigation={onClose}
+                  gotowelcomewhennotenrolled="true"
```

Comment:
Any reason this has to be all lower case?

Reply:
Yup no need for that... solved in this commit: https://github.com/OriginProtocol/origin/pull/2681/commits/649dcd41e84e3e3cd05651e04dd14c8819eba302